scala-package/examples/src/main/scala/ml/dmlc/mxnet/examples/neuralstyle/NeuralStyle.scala

```diff
@@ -0,0 +1,246 @@
+package ml.dmlc.mxnet.examples.neuralstyle
+
+import ml.dmlc.mxnet._
+import org.kohsuke.args4j.{CmdLineParser, Option}
+import org.slf4j.LoggerFactory
+import scala.collection.JavaConverters._
+import com.sksamuel.scrimage.Image
+import java.io.File
+import com.sksamuel.scrimage.Pixel
+import com.sksamuel.scrimage.filter.GaussianBlurFilter
+import com.sksamuel.scrimage.nio.JpegWriter
+import ml.dmlc.mxnet.optimizer.SGD
+import ml.dmlc.mxnet.optimizer.Adam
+
+object NeuralStyle {
+  case class Executor(executor: ml.dmlc.mxnet.Executor, data: NDArray, dataGrad: NDArray)
```

Comment:
better to make it another name, like 'NSExecutor', otherwise it is somewhat confusing